app/domains.js

```diff
@@ -46,7 +46,8 @@ var approvedDomains = [
   'stfc.ac.uk',
   'tnlcommunityfund.org.uk',
   'wiltonpark.org.uk',
-  'ukri.org'
+  'ukri.org',
+  'hes.scot'
```

Comment:
The list should be in (mostly) alphabetical order

Reply:
Sounds good, let me tweak the PR.